src/ast/semantic_analyser.cpp

```diff
@@ -1296,17 +1296,17 @@ bool SemanticAnalyser::check_arg(const Call &call, Type type, int arg_num, bool
   return true;
 }
 
-bool SemanticAnalyser::check_alpha_numeric(const Call &call, int arg_num __attribute__((unused)))
+bool SemanticAnalyser::check_symbol(const Call &call, int arg_num __attribute__((unused)))
 {
   if (!call.vargs)
     return false;
 
   auto &arg = static_cast<String&>(*call.vargs->at(0)).str;
 
-  bool is_alpha = std::regex_match(arg, std::regex("^[a-zA-Z0-9_-]+$"));
+  bool is_alpha = std::regex_match(arg, std::regex("^[a-zA-Z0-9./_-]+$"));
   if (!is_alpha)
   {
-    err_ << call.func << "() expects an alpha numeric string as input";
+    err_ << call.func << "() expects a string that is a valid symbol as input [a-zA-Z0-9./_-]";
```

Comment:
Should we hoist the regex out into a variable so we don't have to maintain two copies of the regex?